libs/plugins/google-analytics/src/lib/plugins-google-analytics.ts

```diff
@@ -0,0 +1,26 @@
+import { registerPlugin, getMyConfig } from '@scullyio/scully';
+
+export const GoogleAnalytics = 'googleAnalytics';
+
+export const googleAnalyticsPlugin = async (html: string): Promise<string> => {
+  const googleAnalyticsConfig = getMyConfig(googleAnalyticsPlugin);
+
+  if (!googleAnalyticsConfig) {
+    throw new Error('googleAnalytics plugin missing Global Site Tag');
+  }
+
+  const googleAnalyticsScript = `
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-171495765-1"></script>
```

Comment:
I believe the query param should be the users analytics id as well.

Reply:
@GuzmanPI Can you check this ??? Thanks @sburke56

Reply:
I have checked this and everything is fine